.github/workflows/changesets.yml

```diff
@@ -3,9 +3,6 @@ on:
     push:
         branches:
             - main
-env:
-    CI: true
-    PNPM_CACHE_FOLDER: .pnpm-store
```

Comment:
Removed these as `PNPM_CACHE_FOLDER` is no longer necessary and `CI` is already an applied env var (see: https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables)